index.html

```diff
@@ -812,6 +891,33 @@ <h2>Generate a network error report</h2>
           </dl>
         </li>
 
+        <li>
+          If <var>request body</var>'s <code>server_ip</code> property is
+          non-empty, and not equal to <var>policy</var>'s <a>received IP
+            address</a>:
+
+          <ol>
+            <li>
+              Set <var>request body</var>'s <code>phase</code> to

```

Comment:
I might express this as completely overwriting _report body_ instead of clearing particular fields. That way, if someone in the future adds sensitive fields but forgets to update this step, the algorithm will fail closed instead of open. (This might just be temporary paranoia on my part though.)

Reply:
There's now some text down in Privacy Considerations that calls out what kind of information can be included in a report, so instead of replacing *report body* I added a step that user agents must clear out any fields that are derived from information that isn't available during DNS resolution.